src/qml/pages/EditProfilePage.qml

```diff
@@ -0,0 +1,188 @@
+                scale: 1
+            }
+            PropertyChanges {
+                target: confirmForm
+                opacity: 0
+                enabled: false
+                scale: 0.8
+            }
+        },
+        State {
+            name: "phone confirmation"
+            PropertyChanges {
+                target: profileForm
+                opacity: 0
+                enabled: false
+                scale: 1.2
+            }
+            PropertyChanges {
+                target: confirmForm
+                opacity: 1
+                enabled: true
+                scale: 1
+            }
+        },
+        State {
+            name: "email confirmation"
+            PropertyChanges {
+                target: profileForm
+                opacity: 0
+                enabled: false
+                scale: 1.2
+            }
+            PropertyChanges {
+                target: confirmForm
+                opacity: 1
+                enabled: true
+                scale: 1
+            }
+        }
+    ]
+
+    transitions: Transition {
+        NumberAnimation {properties: "opacity, scale, enabled"; duration: Theme.animationDuration; easing.type: Easing.InExpo}
+    }
+
+
+    Form {
+        id: profileForm
+
+        Avatar {
+            Layout.alignment: Qt.AlignHCenter
+            diameter: 80
+            nickname: appState.userId
+        }
+
+        Label {
+            Layout.alignment: Qt.AlignHCenter
+            Layout.bottomMargin: 50
+            font.pointSize: UiHelper.fixFontSz(18)
+            color: Theme.primaryTextColor
+            text: appState.userId
+        }
+
+        FormInput {
+            id: phone
+            label: isPhoneConfirmed ? qsTr("Confirmed phone") : qsTr("Phone")
+            password: false
+            placeholder: qsTr("Enter phone")
+            text: appState.phoneNumber
+            onTextChanged: appState.phoneNumber = text
+            validator: RegExpValidator { regExp: /^[\+]?[(]?[0-9]{3}[)]?[-\s\.]?[0-9]{3}[-\s\.]?[0-9]{4,6}$/im }
+            enabled: !isPhoneConfirmed
+        }
+
+        FormPrimaryButton {
+            text: isPhoneConfirmed ? qsTr("Reset") : qsTr("Confirm")
+            enabled: phone.acceptableInput
+            onClicked: phoneButtonClicked(isPhoneConfirmed)
+        }
+
+        FormInput {
+            id: email
+            label: isEmailConfirmed ? qsTr("Confirmed email") : qsTr("Email")
+            password: false
+            placeholder: qsTr("Enter email")
+            text: appState.email
+            onTextChanged: appState.email = text
+            validator: RegExpValidator { regExp:/\w+([-+.']\w+)*@\w+([-.]\w+)*\.\w+([-.]\w+)*/ }
+            enabled: !isEmailConfirmed
+        }
+
+        FormPrimaryButton {
+            text: isEmailConfirmed ? qsTr("Reset") : qsTr("Confirm")
+            enabled: email.acceptableInput
+            onClicked: emailButtonClicked(isEmailConfirmed)
+        }
+    }
+
+    Form {
```

Comment:
We agreed to have different pages/states for profile and confirmation code

Reply:
Second version of the code has all the corrections